common/src/main/java/com/wynntils/features/inventory/ItemTextOverlayFeature.java

```diff
@@ -237,12 +237,10 @@ public TextOverlay getTextOverlay() {
             String text = dungeon.getInitials();
 
             CustomColor textColor;
-            if (dungeon.isRemoved()) {
-                textColor = REMOVED_COLOR;
-            } else if (item.isCorrupted()) {
-                textColor = CORRUPTED_COLOR;
+            if (item.isCorrupted()) {
+                textColor = dungeon.isCorruptedRemoved() ? REMOVED_COLOR : CORRUPTED_COLOR;
             } else {
-                textColor = STANDARD_COLOR;
+                textColor = dungeon.isRemoved() ? REMOVED_COLOR : STANDARD_COLOR;
             }
```

Comment:
Perhaps it would be cleaner if you follow this pseudo code.

```
if item is removed
then return removed color

return iscorrupted ? corruptedcolor : normalcolor
```

Reply:
Nevermind.., I just saw the change..